images/bootstrap/runner

```diff
@@ -16,31 +16,48 @@
 set -o errexit
 set -o nounset
 set -o pipefail
-set -x
```

Comment:
(assume we still want to trace the bootstrap call?)

Reply:
TBH it turned out to be noisier than it's worth I think, compare the job we were just looking at to the new log snippet above. With trace on you get weird things in the logs that are just more noise mostly. 

I'm fairly confident we can rely on knowing the image tag to know what a build log was doing for tracing.